src/components/Votes.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import { patchVotes } from "./utils";
+
+const Votes = ({ votes, article_id }) => {
+  const [currentVotes, setCurrentVotes] = useState(votes);
+
+  const userVote = (inc) => {
+    setCurrentVotes(currentVotes + inc)
```

Comment:
Instead of using currentVotes from the state I'd put a function in there so that you can gain access to the currentVotes.